<launch>
  <arg name="typeColormap" default="2" />
  <arg name="typeLepton" default="3" />
  <arg name="spiSpeed" default="20" />
  <arg name="rangeMin" default="-1" />
  <arg name="rangeMax" default="-1" />
  <arg name="loglevel" default="0" />
  <arg name="autoScale" default="1" />

  <node pkg="flir_lepton" type="flir_lepton_node" name="flir_lepton" output="screen">
    <param name="typeColormap" type="int" value="$(arg typeColormap)" />
    <param name="typeLepton" type="int" value="$(arg typeLepton)" />
    <param name="spiSpeed" type="int" value="$(arg spiSpeed)" />
    <param name="rangeMin" type="int" value="$(arg rangeMin)" />
    <param name="rangeMax" type="int" value="$(arg rangeMax)" />
	  <param name="loglevel" type="int" value="$(arg loglevel)" />
    <param name="autoScale" type="int" value="$(arg autoScale)" />
  </node>
</launch>
